tests/e2e/e2e_test.go

```diff
@@ -18,8 +18,6 @@ func (s *IntegrationTestSuite) TestGov() {
 
 // globalfee in genesis is set to be "0.00001uatom"
 func (s *IntegrationTestSuite) TestQueryGlobalFeesInGenesis() {
```

Comment:
Getting a failure on the zero coin fee...`feeInGenesis` is ` 0.000010000000000000uatom`, and `fees` is `0.000000000000000000photon,0.000001000000000000uatom`

Reply:
this is due to the globalfee last proposal of changing globalfee back to original fee failed, and query globalfee is after global fee test.